templates/internet-of-things/appstore.html

```diff
@@ -332,11 +345,13 @@ <h3 class="p-card__title">Snap Store Proxy</h3>
 <section class="p-strip--light">
   <div class="row">
     <div class="col-8">
-      <h2>Dedicated App Store</h2>
-      <h3 class="p-heading--four">Curate your own app collection with custom pricing and unique applications.</h3>
-      <p>Having a managed app store allows you role-based access controls, complete control of application versions, updates and controlled rollouts for $15,000 per year. We enable integrators and VARs to operate app stores on behalf of their clients. Access to the full Ubuntu app ecosystem requires device certification.</p>
+      <h2>Global Snap Store</h2>
+      <h3 class="p-heading--four">Showcase your applications on the IoT marketplace for Linux.</h3>
+      <p>The Snap Store is the global marketplace for applications on Linux, used by millions of people around the world. Discover snaps, a unique packaging format, optimised for OTA updates and security, perfect for IoT, desktop and cloud. With a huge range of applications available and free to use, find the software you need for your use case. Showcase your applications and boost your user base.</p>
     </div>
-    <a href="/certified">Check if your device is certified&nbsp;&rsaquo;</a>
+    <p>
+      <a href="/certified" class="p-button--positive">Discover the Snap Store&nbsp;&rsaquo;</a>
```

Comment:
This link should be to https://snapcraft.io/

Reply:
Done thanks!